src/style.css

```diff
@@ -1,3 +1,79 @@
+html {
+  font-family: Roboto, sans-serif;
+  font-weight: 500;
+  margin: 0;
+}
+
 body {
   margin: 0;
 }
+
+.header {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  padding: 0 50px;
+  box-sizing: border-box;
+  width: 100%;
+  align-items: center;
+}
+
+.logo {
+  width: 40px;
+  height: 40px;
+  display: flex;
+  align-items: center;
+}
+
+.nav__item {
+  list-style-type: none;
+  display: flex;
+  align-items: center;
+  margin: 0 0 0 20px;
+}
```

Comment:
It's recommended to be consistent with your margins. In this case, you're adding margin to the left side of the `.nav__item` class. It would be better to add only top or bottom margin, but not both. [Checklist: Be consistent with your *vertical* margins]